### PR TITLE
Milestone: Finished Working UX Forms

### PR DESCRIPTION
**Pull request by BlakeYall**
In this Milestone, I have done the following step to set up a "My-Portfolio" repository.

- [ ] Updated the contact form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -226,6 +226,72 @@ <h3><span>&#9679;</span>Skills</h3>
...
+      <section>
+        <div class="ux-form1">
+          <div class="contact-info">
+            <h3>
+              I'm always interested in hearing about new projects, so if you'd
+              like to chat please get in touch.
+            </h3>
+          </div>
+          <div class="contact-form">
+            <form action="/login" method="post" class="user-form">
```
Comment (levy002) [inline index.html:238]: - [ ] Kindly make sure when your form is submitted the data is collected by using the [Formspree service](https://formspree.io/html).

File: index.html [hunk 2]
```diff
@@ -226,6 +226,72 @@ <h3><span>&#9679;</span>Skills</h3>
...
+              <ul>
+                <li>
+                  <label for="name">Full Name</label>
+                  <input
+                    type="text"
+                    id="name"
+                    name="user-name"
+                    placeholder="John Doe"
+                  />
```
Comment (levy002) [inline index.html:247]: - [ ] Kindly make sure to add the `required` attribute which is a built-in HTML validation for preventing empty submission. :wink: 
- [ ] Kindly make sure to validate your name field to accept up to 30 chars. :wink:

File: index.html [hunk 3]
```diff
@@ -226,6 +226,72 @@ <h3><span>&#9679;</span>Skills</h3>
...
+                </li>
+                <li>
+                  <label for="email">Email address:</label>
+                  <input
+                    type="email"
+                    id="email"
+                    name="user-email"
+                    placeholder="johndoe@gmail.com"
+                  />
```
Comment (levy002) [inline index.html:256]: - [ ] Kindly make sure to add the `required` attribute which is a built-in HTML validation for preventing empty submission. :wink:

File: index.html [hunk 4]
```diff
@@ -226,6 +226,72 @@ <h3><span>&#9679;</span>Skills</h3>
...
+                </li>
+                <li>
+                  <label for="msg"></label>
+                  <textarea
+                    id="msg"
+                    name="user-message"
+                    placeholder="Enter text here"
+                  ></textarea>
```
Comment (levy002) [inline index.html:264]: - [ ]  Kindly make sure to validate your message field to accept up to 500 chars. :wink:

File: styles.css [hunk 5]
```diff
@@ -590,3 +631,101 @@ hr {
     text-align: center;
   }
 }
+
+.ux-form1 {
+  display: flex;
+  flex-direction: row;
+}
```
Comment (levy002) [inline styles.css:638]: - [ ] Kindly make sure to align your contact section contacts correctly like on the provided Figma design. :wink: 

   - ### Your design
   image


  - ### Figma design
   image

File: styles.css [hunk 6]
```diff
@@ -590,3 +631,101 @@ hr {
...
+
+input:focus,
+textarea:focus {
+  border: #333;
+}
+
+.button {
+  padding-left: 90px;
+}
```
Comment (levy002) [inline styles.css:678]: - [ ] Kindly make sure to stick to the provided Figma while styling the `form`. :wink: 

   - ### Your design

   image

  - ### Figma design
   image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @prg-04   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -226,6 +226,71 @@ <h3><span>&#9679;</span>Skills</h3>
...
+                    name="user-name"
+                    maxlength="30"
+                    required
+                  />
+                </label>
+                <label>
+                  Email address
+                  <input type="email" id="email" name="user-email" required />
+                </label>
+                <label>
```
Comment (mahbubul14) [inline index.html:258]: - [ ] Kindly add placeholder text for the name and email address to match the Figma design. :+1:

File: index.html [hunk 8]
```diff
@@ -226,6 +226,71 @@ <h3><span>&#9679;</span>Skills</h3>
...
+                  <textarea
+                    id="msg"
+                    name="user-message"
+                    placeholder="Enter text here"
+                    minlength="10"
+                    maxlength="500"
+                  ></textarea>
+                </label>
+                <button type="submit">Get in touch</button>
+              </fieldset>
```
Comment (mahbubul14) [inline index.html:268]: - [ ] Kindly remove the grey line from the border to match the Figma design. You can use "border: none" for it. :+1:

File: index.html [hunk 9]
```diff
@@ -226,6 +226,71 @@ <h3><span>&#9679;</span>Skills</h3>
...
+              <a href="#"><i class="fa-brands fa-github"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-solid fa-m"></i></a>
```
Comment (mahbubul14) [inline index.html:289]: - [ ] Kindly decrease the size of your social icon to match the Figma design. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @BlakeYall ,

Great work on making the changes requested by the previous reviewer 👏

You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Proper Github flow used.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 8 above)
Comment (prg-04) [reply index.html:268]: The gray line is included in the figma design please check

**Review by emmiiorji — APPROVED** [review]
Hi @prg-04, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations


## Good Highlights ✅

- Good commit messages
- Implemented requested changes
- Adherence to GitHub flow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**